ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/components/CustomPositionIndicator.kt

```diff
@@ -38,12 +39,22 @@ fun CustomPositionIndicator(
 }
 
 @Preview(
-    widthDp = 300,
-    heightDp = 300,
     apiLevel = 26,
     uiMode = Configuration.UI_MODE_TYPE_WATCH,
-    backgroundColor = 0x000000,
-    showBackground = true
+    showSystemUi = true,
+    device = Devices.WEAR_OS_LARGE_ROUND
+)
+@Preview(
+    apiLevel = 26,
+    uiMode = Configuration.UI_MODE_TYPE_WATCH,
+    showSystemUi = true,
+    device = Devices.WEAR_OS_SQUARE
+)
+@Preview(
+    apiLevel = 26,
+    uiMode = Configuration.UI_MODE_TYPE_WATCH,
+    showSystemUi = true,
+    device = Devices.WEAR_OS_SMALL_ROUND
```

Comment:
Note: this requires the Android Studio Canary.  Is it safe to assume that given we are on Alpha Compose versions?

We could also (probably) warn in the the gradle build if you are using an older Android Studio.

Reply:
A crude-ish way to force that would be to update the AGP to the `7.2.x` alpha corresponding to Chipmunk.

That's not super friendly though, as most people would probably be on stable.

Reply:
It shouldn't break the build, but the preview won't be clean, but getting developers familiar with this syntax sounds nice.

Reply:
I think a comment next to the `device` field with "This will only be rendered properly in AS Chipmunk and beyond" would be good to call that out while still making it easy to sync and build the project with the current stable AS.

Once Chipmunk becomes stable we can remove all of those then.